rust/cardano-chain-follower/examples/follow_chains.rs

```diff
@@ -96,7 +96,11 @@ async fn start_sync_for(network: &Network, matches: ArgMatches) -> Result<(), Bo
     let mithril_dl_workers = format!("{}", dl_config.workers);
 
     if let Some(chunk_size) = matches.get_one::<u16>("mithril-sync-chunk-size") {
-        dl_config = dl_config.with_chunk_size(*chunk_size as usize * 1024 * 1024);
+        let chunk_size = (*chunk_size as usize)
+            .checked_mul(1024)
```

Comment:
We can multiply once on the `1 048 576`, but left a comment that its a `1024 * 1024`.